operations/mimir-mixin/config.libsonnet

```diff
@@ -51,6 +51,13 @@
     job_labels: [$._config.per_cluster_label, 'namespace', 'job'],
     cluster_labels: [$._config.per_cluster_label, 'namespace'],
 
+    // PromQL queries used to find clusters and namespaces with Mimir.
+    dropdown_queries: {
```

Comment:
Have you considered calling it "dashboard_variables" to match the naming in Grafana?

Reply:
These are not variables, but PromQL queries to select values for those dashboard variables. Would `dashboard_variables` fit?

Reply:
To me it's still configuration for the dashboard variables, but I don't feel strong about it :) I already approved this PR :)